### ISSUE TITLE
Course Archival with Scores & Grading

### ISSUE THREAD

**Issue by dfuchss** [ls1intum/Artemis#5081]
### Describe the solution you'd like

When archiving a course currently the ZIP does not include all necessary information (regarding grading)
e.g., the tutor-comments for the tasks are missing.

Since we have to remove the data from the students after a specific time from the server, we want to archive all necessary information regarding the grading (Exercises & Exam Points & Comments & Complaints)

**Comment by dfuchss** [comment]
I still think the export should at least include the scoring. without scores, we would miss important information for the grading

**Comment by krusche** [comment]
I agree, we should work on this asap and avoid having an issue open without any activity for 3 years

**Comment by dfuchss** [comment]
Hi there! 👋

I wanted to gently check in on this issue since it's been a while since the last update. Are there any plans to work on this, or is there anything I can do to help move it forward? 😊

Thank you for your time and consideration! 🙏

**Comment by kaleliguray** [comment]
Linked issue is still open, thus assuming this issue is also still relevant

**Comment by krusche** [comment]
#11885 improved the course archival with course and exam scores. Please try it out and let us know if you are still missing specific data in the course archival
